<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLCDNumber" name="lcd_sec">
   <property name="geometry">
    <rect>
     <x>845</x>
     <y>20</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_min">
   <property name="geometry">
    <rect>
     <x>765</x>
     <y>20</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>100</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 italic 22pt &quot;Cascadia Mono SemiBold&quot;;</string>
   </property>
   <property name="text">
    <string>BLACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="lastOneLabel">
   <property name="geometry">
    <rect>
     <x>845</x>
     <y>175</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>LastOne: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="IPEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>140</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>10.47.145.75</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="reConnectButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>175</y>
     <width>88</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Reconnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SREJECT_OP">
   <property name="geometry">
    <rect>
     <x>965</x>
     <y>383</y>
     <width>108</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>REJECT_OP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverGetEdit">
   <property name="geometry">
    <rect>
     <x>845</x>
     <y>210</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="SLEAVE_OP">
   <property name="geometry">
    <rect>
     <x>965</x>
     <y>418</y>
     <width>108</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>LEAVE_OP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getButton_1">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>210</y>
     <width>88</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>CilentGet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SMOVE_OP">
   <property name="geometry">
    <rect>
     <x>965</x>
     <y>453</y>
     <width>108</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>MOVE_OP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverSendEdit">
   <property name="geometry">
    <rect>
     <x>845</x>
     <y>279</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="clientSend">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>314</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="reSetButton">
   <property name="geometry">
    <rect>
     <x>878</x>
     <y>139</y>
     <width>80</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>ReSet</string>
   </property>
  </widget>
  <widget class="QLabel" name="connectLabel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>175</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Connection fail</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="CREJECT_OP">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>383</y>
     <width>88</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>REJECT_OP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showClientButton">
   <property name="geometry">
    <rect>
     <x>965</x>
     <y>139</y>
     <width>108</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>ShowClients</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CLEAVE_OP">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>418</y>
     <width>88</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>LEAVE_OP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getButton_2">
   <property name="geometry">
    <rect>
     <x>965</x>
     <y>210</y>
     <width>108</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>ServerGet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reStartButton">
   <property name="geometry">
    <rect>
     <x>965</x>
     <y>175</y>
     <width>108</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>RestartServer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CGG_OP">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>488</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>GG_OP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverSend">
   <property name="geometry">
    <rect>
     <x>845</x>
     <y>314</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="PORTEdit">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>140</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>16667</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="clientSendEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>279</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="serverGet">
   <property name="geometry">
    <rect>
     <x>845</x>
     <y>245</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="CilentGiveup">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>524</y>
     <width>88</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>GiveUp</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SREADY_OP">
   <property name="geometry">
    <rect>
     <x>965</x>
     <y>348</y>
     <width>108</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>READY_OP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SGG_OP">
   <property name="geometry">
    <rect>
     <x>965</x>
     <y>488</y>
     <width>108</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>GG_OP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="serverSendButton">
   <property name="geometry">
    <rect>
     <x>965</x>
     <y>279</y>
     <width>108</width>
     <height>28</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ServerSend</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CREADY_OP">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>348</y>
     <width>88</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>READY_OP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ServerGiveup_2">
   <property name="geometry">
    <rect>
     <x>965</x>
     <y>524</y>
     <width>108</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>GiveUp</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="clientGet">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>245</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="CMOVE_OP">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>453</y>
     <width>88</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>MOVE_OP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clientSendButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>279</y>
     <width>88</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>ClientSend</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="clientGetEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>210</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
